actions/structures.xml

```diff
@@ -30,19 +30,10 @@
                     <action id="item-authoring" name="Authoring" url="/taoItems/Items/authoring" group="content" context="instance" binding="launchEditor">
                         <icon id="icon-edit"/>
                     </action>
-                    <action id="item-filter" name="Filter" context="*" group="search" binding="filter">
-                        <icon id="icon-filter"/>
-                    </action>
-                    <action id="item-search" name="Search" url="/taoItems/Items/search" context="*" group="search" binding="launchFinder">
-                        <icon id="icon-find"/>
-                    </action>
                     <action id="item-class-new" name="New class" url="/taoItems/Items/addSubClass" context="resource" group="tree" binding="subClass">
                         <icon id="icon-folder-open"/>
                     </action>
-                    <action id="item-delete" name="Delete" url="/taoItems/Items/deleteItem" context="instance" group="tree" binding="removeNode" >
-                        <icon id="icon-bin"/>
-                    </action>
-                    <action id="item-class-delete" name="Delete" url="/taoItems/Items/deleteClass" context="class" group="tree" binding="removeNode" >
+                    <action id="item-delete-all" name="Delete" url="/taoItems/Items/deleteAll" context="resource" multiple="true" group="tree" binding="removeNodes">
```

Comment:
Is the remove of `item-delete` action intentional? It is still linked above (https://github.com/oat-sa/extension-tao-item/pull/294/files#diff-384f0ccf46984d5475abe29b5b92c56cR17).